### ISSUE TITLE
Filters Breaks With Accented Characters

### ISSUE THREAD

**Issue by madranet** [gilbertchen/duplicacy#240]
I have part of my exclusion filter set up as follows:

```
-Documents/Adobe/*
-Documents/Crosswords/*
-Documents/Haiceáil/*
-Documents/Schematics/*
-Documents/Smith Micro/*
+Documents/*
```

When I do a `-dry-run` all the sub-directories under `~/Documents` are uploaded apart from the ones listed for exclusion, which is what I want...  EXCEPT for `Documents/Haiceáil/*` which is also uploaded. 

Obvious conclusion is that the filtering function does not match against accented characters.  I'm going to try it again, replacing the `á` with a ? to see if that sorts it.

**Comment by madranet** [comment]
Hmmm...

Well I ran `duplicacy backup -dry-run -stats >> ~/Desktop/duptest.txt` again  with my filters file amended to read:

```
-Documents/Adobe/*
-Documents/Crosswords/*
-Documents/Haice?il/*
-Documents/Schematics/*
-Documents/Smith Micro/*
+Documents/*
```

..and the `Documents/Haiceáil/` directory was still uploaded.  What am I doing wrong here?

**Comment by gilbertchen** [comment]
`-Documents/Haice.il/*` should work.

**Comment by madranet** [comment]
Nope. A dot didn't work either.

BTW. I presume the dot is what's supposed to be between the first pair of inverted commas in the docs, but seems to be missing. That did puzzle me:


>...Patterns may contain wildcard characters "" which matches a path string of any length, and "?" matches a single character...

**Comment by gilbertchen** [comment]
Sorry I got confused with the regex wildcards.  Did you try `-Documents/Haice*il/*`?

**Comment by madranet** [comment]
Yes. that seems to work. So I presume that `*` is the missing symbol from the docs [as quoted above]?

Out of interest, why is the `*` needed? Is it because `?` is actually matching against a single byte so doens't match against an accented character, which is double-byte?
